Sources/Build/BuildPlan.swift

```diff
@@ -317,6 +318,20 @@ public final class SwiftTargetDescription {
         return args
     }
 
+    /// A list of compilation conditions to enable for conditional compilation expressions.
+    private var activeCompilationConditions: [String] {
+        var compilationConditions = ["-DSWIFT_PACKAGE"]
+
+        switch buildParameters.configuration {
+        case .debug:
+            compilationConditions += ["-DDEBUG"]
+        default:
```

Comment:
Instead of default, can you add the explicit `.release` case? This is helpful when we're adding new cases to the enum. The compiler will produce an error and we will be forced to rethink this switch case.